### PR TITLE
Term frequency

### PR DESCRIPTION
**Pull request by lintangsutawika**
@haileyschoelkopf some updates.

### CODE REVIEW

File: analysis/term_frequency/task.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def __init__(self, DATASET_NAME):
         )
 
     def doc_to_text(self, doc):
-        return "Q: What is {x1} times {x2}? A:".format(**doc)
+        return "Q:What is {x1} times {x2}? A:".format(**doc)
```
Comment (haileyschoelkopf) [inline analysis/term_frequency/task.py:128]: Which format was used for the paper experiments?

**Review by haileyschoelkopf — APPROVED** [review]
Looks good, feel free to merge! Had one question about the task prompts.

File: analysis/term_frequency/task.py (hunk 1 above)
Comment (lintangsutawika) [reply analysis/term_frequency/task.py:128]: Sorry, I missed this. The latest one is the one used. Without the space.

Comment (haileyschoelkopf) [reply analysis/term_frequency/task.py:128]: nw! gotcha, merging!

**Pull request merged**